### ISSUE TITLE
Are you intended for mapping of osquery on linux attack matrix??

### ISSUE THREAD

**Issue by naina-emmanuel** [teoseller/osquery-attck#1]
Please let me know if osquery has mapped to Linux att&ck matrix already or you planning to map it?

**Comment by teoseller** [comment]
Hi
yes is in planning to create the same for Linux

Thanks

**Comment by naina-emmanuel** [comment]
> Hi
> yes is in planning to create the same for Linux
> 
> Thanks

Great! looking forward to it :). Can we have some estimated time that when we can have the mapped matrix?

**Comment by benbasscom** [comment]
To add on to this, any desire to map for macOS?

**Comment by teoseller** [comment]
Yes i would like to add both macos and linux, after this super busy months i will start to add some conf pack for macos . I hope that the community will add more query .

Stay tuned :)

**Comment by jahshuah** [comment]
Add me to the list of people interested in MacOS and Linux content. Let me look at what it might take to develop some content as well.

**Comment by teoseller** [comment]
yes for sure

**Comment by teoseller** [comment]
Linux add now, please suggestions to improve or add other query are really welcome

**Comment by xiaodupi-zyq** [comment]
When can the author continue to update the query related to the more with windows or linux？
